### ISSUE TITLE
Options for start with windows and auto update check

### ISSUE THREAD

**Issue by Freeman-ab** [fxsound2/fxsound-app#172]
Can you add options to enable or disable in settings,
    ● start with windows
    ● auto or manual update checks

We can already enable or disable them other ways but I think it would be better if you give users options in app.

**Comment by jonolds** [comment]
agreed, forcing automatic update checks is unacceptable for a utility that otherwise shouldn't use the internet.

**Comment by bvijay74** [comment]
Automatic update is done through Task Scheduler. You can find an FxSound/Update task in Task Scheduler which checks for updates. You can delete it manually if you do not want automatic update check.
We can consider adding an option in the application to disable automatic update check.

**Comment by raddyst** [comment]
> We can consider adding an option in the application to disable automatic update check.

Yes, better with this option than without it.

**Comment by bvijay74** [comment]
Auto updates can be enabled or disabled from settings from v1.2.2.0. 
To manually check for updates, use the check for updates menu option.
